### PR TITLE
Task #119686 feat: Write Widgets Service(Correct the widget json)

### PR DESCRIPTION
**Pull request by sudhir-j-sapkal**

### CODE REVIEW

File: code/src/app/unitenew/configs/widgets.config.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+@Injectable()
+export class Widgets{
+    
+    constructor(private _httpClient : HttpClient)
+    {
+    }
+
+    getWidgets() {
+        return this._httpClient.get("/assets/widgets.json").map(data => {
+            return data;
```
Comment (sushan05) [inline code/src/app/unitenew/configs/widgets.config.ts:13]: @ttplsudhir  `map` is not needed here.

File: code/src/app/unitenew/unite.module.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { AdminBranch } from './branch/admin/admin.branch';
 
 import { GlobalConfig } from './configs/global.configs';
 import { Menues } from './configs/menus.configs';
```
Comment (sushan05) [inline code/src/app/unitenew/unite.module.ts:11]: @ttplsudhir - Use correct spelling. It should be `Menus`

File: code/src/app/unitenew/uniteServices/routingService.ts [hunk 3]
```diff
@@ -170,4 +174,37 @@ export class UniteRouting{
...
+    getAllWidgets()
+    {
+        this._widget.getWidgets().subscribe(data =>{
+            this.mapWidgetsWithPages(data);
+        });
+    }
+
+    mapWidgetsWithPages(widgets)
+    {
+        let i = 0;
```
Comment (sushan05) [inline code/src/app/unitenew/uniteServices/routingService.ts:187]: @ttplsudhir : No need of explicit incremental variable for indexing. `forEach` does provides an index parameter. Check out this - https://www.w3schools.com/jsref/jsref_forEach.asp

Comment (sudhir-j-sapkal) [reply code/src/app/unitenew/uniteServices/routingService.ts:187]: @sushan05 Yes will update the code accordingly

File: code/src/app/unitenew/unite.module.ts (hunk 2 above)
Comment (sudhir-j-sapkal) [reply code/src/app/unitenew/unite.module.ts:11]: @sushan05 Yeah will update code accordingly

File: code/src/app/unitenew/configs/widgets.config.ts (hunk 1 above)
Comment (sudhir-j-sapkal) [reply code/src/app/unitenew/configs/widgets.config.ts:13]: @sushan05 Ok will check this and update code accordingly

**Pull request merged**